tests/switchboardOnDemand.ts

```diff
@@ -0,0 +1,172 @@
+import * as anchor from '@coral-xyz/anchor';
```

Comment:
is this actually used? delete?

Reply:
yeah when initting the context, need it to define a workspace